lib/lte_link_control/lte_lc_modem_hooks.c

```diff
@@ -76,6 +76,14 @@ static void on_modem_init(int err, void *ctx)
 		(void)lte_lc_proprietary_psm_req(false);
 	}
 
+	/* Only supported in mfw 2.0.1 and newer.
+	 * Ignore the return value; an error likely means that the feature
+	 * is not supported. This optimization was enabled by default in older firmware,
```

Comment:
I guess this was only enabled by default in MFW v2.0.0, not in v1.x which probably don't support the whole feature.

Reply:
ok, rephrased